lib/defaults.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  probotUsername: (process.env.APP_NAME || 'probot-snooze') + '[bot]',
```

Comment:
Workaround until #12 is implemented.

Reply:
@jbjonesjr oh, I forgot, this was the actual motivation for making this change. The bot name needs to be configurable until #12 is fixed. Would you reconsider this PR for that reason?